src/core_codemods/order_imports.py

```diff
@@ -14,7 +14,7 @@ class OrderImports(SimpleCodemod):
         name="order-imports",
         summary="Order Imports",
         review_guidance=ReviewGuidance.MERGE_WITHOUT_REVIEW,
-        has_description=False,
+        description="",
```

Comment:
Not the biggest deal but I'm not in love with `""` and `None` as the sentinels for these cases. I think `has_description` is clearer.

Reply:
This change was necessary as a means to define the description of a sonar codemod in terms of other codemod. As it was, a codemod would only source it's description from a file or not. Thus, I would have to copy-paste the description to it's own file.

Reply:
I understand, I guess what I'm saying is that keeping `has_description` as a way to differentiate those cases would retain some clarity. But it's not a big deal either way.